tap-schema/build-all

```diff
@@ -7,4 +7,5 @@
 ./build idfprod ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
 ./build idfint ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
 ./build idfdev ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build usdf ../yml/live_obscore.yaml
+./build usdf-dev ../yml/dp02_dc2.yaml
```

Comment:
This one needs dp02_obscore.yaml included as well, on the theory that the initial "live Obscore" will be served at "oga.ObsCore" and not "ivoa.ObsCore".